code-challenges.test.js

```diff
@@ -14,38 +14,143 @@
 
 // a) Create a test with an expect statement using the variable provided.
 
-const people = [
-  { name: "ford prefect", occupation: "a hitchhiker" },
-  { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
-  { name: "arthur dent", occupation: "a radio employee" }
-]
-// Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
-
+describe("profileArray", () => {
```

Comment:
Thought I know it is for the sake of web development, I miss the long function names. Great structure